open_bus_gtfs_etl/gtfs_stat/aggregations.py

```diff
@@ -53,10 +53,11 @@ def trip_stats_aggregation(group):
 
         d['all_stop_latlon'] = ';'.join(str(x) + ',' + str(y) for x, y in
                                         zip(group['stop_lat'].tolist(), group['stop_lon'].tolist()))
-
-        for key in keys_for_all:
-            d[f'all_{key}'] = ';'.join(group[key].tolist())
-
+        try:
+            for key in keys_for_all:
+                d[f'all_{key}'] = ';'.join([str(i) for i in group[key].tolist()])
+        except TypeError as e:
+            print(e)
```

Comment:
please add a comment explaining why this exception can be ignored

Reply:
@AvivSela PR #13 was merged, please update this PR

Reply:
the try catch needs to be removed
also, PR #13 was merged, so you can update from main branch and remove this code